base/compiler/types.jl

```diff
@@ -78,14 +78,20 @@ function Effects(e::Effects = EFFECTS_UNKNOWN;
         inbounds_taints_consistency)
 end
 
+is_consistent(effects::Effects)  = effects.consistent === ALWAYS_TRUE
+is_effect_free(effects::Effects) = effects.effect_free === ALWAYS_TRUE
+is_nothrow(effects::Effects)     = effects.nothrow === ALWAYS_TRUE
+is_terminating(effects::Effects) = effects.terminates === ALWAYS_TRUE
```

Comment:
I really don't think this is a good name... Any suggestions?

Reply:
I'd probably just keep the scheme the same as with the other ones and go with `is_terminates`, even though it sounds a little odd

Reply:
Thanks, that sounds reasonable.